kubernetes/charts/weblogic-operator/templates/_operator-clusterrole-namespace.tpl

```diff
@@ -34,4 +34,7 @@ rules:
 - apiGroups: ["batch"]
   resources: ["jobs"]
   verbs: ["get", "list", "watch", "create", "update", "patch", "delete", "deletecollection"]
+- apiGroups: ["policy"]
```

Comment:
I'm surprised that "policy" worked. I would have thought that this would need to be "policy/v1beta1", but apparently it does work :)

Reply:
yes. I initially tried with "policy/v1beta1" and surprisingly that didn't work. After listing the api resources, I saw apiGroup for poddisruptionbudgets is `policy` . Thanks.

```
$  kubectl api-resources | grep poddisruptionbudgets
NAME                               SHORTNAMES   APIGROUP                       NAMESPACED   KIND
poddisruptionbudgets     pdb                    policy                                true                   PodDisruptionBudget
```